packages/camera/camera/example/integration_test/camera_test.dart

```diff
@@ -155,10 +164,9 @@ void main() {
       return;
     }
 
-    final CameraController controller = CameraController(
+    final CameraController controller = CameraController.withSettings(
       cameras[0],
-      ResolutionPreset.low,
-      enableAudio: false,
+      mediaSettings: MediaSettings.low(enableAudio: false),
```

Comment:
can this be `MediaSettings(resolutionPreset: ..., enableAudio: ...)`? It's hard to tell what `low` means from caller

Reply:
Removed `MediaSettings.low()` helper everywhere.